test/test_ops.jl

```diff
@@ -251,5 +251,20 @@ end # of testsection
     @test rotate(a, b, π) ≈ rotate(a, b, -π)
     @test rotate(a, b, π/2) ≈ rotate(a, -b, -π/2)
 end
+
+@testsection "tovoigt/fromvoigt" begin
```

Comment:
Could you try this for more than dimension `3` i.e. by adding a loop `for dim in 1:3` and have `A = rand(Tensor{4,dim})` etc?